mods/persistence/game/objects/structures/window.dm

```diff
@@ -7,4 +7,13 @@
 	set_dir(dir)
 	if(is_fulltile())
 		layer = FULL_WINDOW_LAYER
-	return INITIALIZE_HINT_LATELOAD
\ No newline at end of file
+	return INITIALIZE_HINT_LATELOAD
+
+/obj/structure/window/attackby(obj/item/W, mob/user)
+	// The user is trying to deconstruct the window, so check for permissions.
+	if(istype(W, /obj/item/gun/energy/plasmacutter) || isCrowbar(W) || isScrewdriver(W) || isWrench(W))
+		if(!check_area_protection(user))
+			to_chat(user, SPAN_DANGER("A magnetic force repels your attempt to deconstruct \the [src]!"))
```

Comment:
Not to diss the power of magnets or anything, but shouldn't this be a "gravitic" or "gravitational"? While "gravitic" is a rarely used term compared to gravitational, it'd probably fit into gameplay terminology better.

Reply:
The idea here is mostly that the magnetic field is preventing you from using precision deconstruction techniques like unscrewing the screws or unwrenching the bolts. You can still slam something heavy into the wall to damage it, which presumably a gravitational field would be able to easily prevent.

Either way, it doesn't make a whole lot of sense why this works, but it at least doesn't stretch *too* far beyond the in game technology that already exists. The implications of being able to control gravitational forces that precisely and relatively cheaply creates more questions than it answers, in my opinion.